src/containers/UserSettings/settings.tsx

```diff
@@ -143,7 +141,7 @@ export const autocompleteOnEnterSetting: SettingProps = {
 export const interfaceVersionInfoField: SettingsInfoFieldProps = {
     title: i18n('settings.about.interfaceVersionInfoField.title'),
     type: 'info',
-    content: packageJson.version,
+    content: process.env.UI_VERSION,
```

Comment:
I believe process.env.UI_VERSION may be unavailable at client side
mb should write to window in index.html